app/views/orga/teams/show.html.slim

```diff
@@ -79,7 +79,7 @@ nav.btn-group
   ul.conferences
     - @team.members.each do |user|
       - if user.conferences.in_current_season.any?
-        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season, true).join(',')}".html_safe
+        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season).join(',')}".html_safe
```

Comment:
I totally agreed, but I didn't find a way to solve it. Spent too much time on it, I'll make a note and will try it another time. Okay?